src/frontend/gatsby/src/components/Post/Post.jsx

```diff
@@ -101,7 +101,11 @@ const Post = ({ postUrl }) => {
   // We need a ref to our post content, which we inject into a <section> below.
   const sectionEl = useRef(null);
   // Grab the post data from our backend so we can render it
-  const { data: post, error } = useSWR(postUrl, (url) => fetch(url).then((r) => r.json()));
+  const { data: post, error } = useSWR(postUrl, async (url) => {
```

Comment:
You can simplify this, and use `useSWR(postUrl)` with no second arg (the default fetcher will get used).